tests/tensorflow/sota_checkpoints_eval.json

```diff
@@ -6,253 +6,254 @@
                     "target_init": 77.9,
-                    "resume": "inception_v3_sparsity_54",
+                    "resume": "inception_v3_imagenet_magnitude_sparsity",
                     "metric_type": "Acc@1",
-                    "model_description": "inception_v3_sparsity_54",
-                    "compression_description": "Sparsity 54",
+                    "model_description": "Inception V3",
+                    "compression_description": "Sparsity 54% (Magnitude)",
                     "reverse_input_channels": true,
                     "mean_value": "[127.5,127.5,127.5]",
                     "scale_value": "[127.5]",
                     "diff_fp32_min": -1,
                     "diff_fp32_max": 0.1
                 },
-                "mobilenet_v2": {
+                "mobilenet_v2_imagenet": {
                     "config": "tests/tensorflow/data/configs/classification/mobilenet_v2_imagenet.json",
                     "target": 71.85,
                     "metric_type": "Acc@1",
-                    "model_description": "mobilenet_v2",
+                    "model_description": "MobileNet V2",
                     "reverse_input_channels": true,
                     "mean_value": "[127.5,127.5,127.5]",
                     "scale_value": "[127.5]"
                 },
-                "mobilenet_v2_int8_w_sym_t_half_a_sym_t": {
+                "mobilenet_v2_imagenet_int8": {
                     "config": "examples/tensorflow/classification/configs/quantization/mobilenet_v2_imagenet_int8.json",
-                    "reference": "mobilenet_v2",
-                    "target": 71.66,
+                    "reference": "mobilenet_v2_imagenet",
+                    "target": 71.63,
                     "target_init": 60.92,
-                    "resume": "mobilenet_v2_int8_w_sym_t_half_a_sym_t",
+                    "resume": "mobilenet_v2_imagenet_int8",
                     "metric_type": "Acc@1",
-                    "model_description": "mobilenet_v2_int8_w_sym_t_half_a_sym_t",
-                    "compression_description": "INT8",
+                    "model_description": "MobileNet V2",
+                    "compression_description": "INT8 (per-tensor, symm. W, asymm. half-range A)",
                     "reverse_input_channels": true,
                     "mean_value": "[127.5,127.5,127.5]",
                     "scale_value": "[127.5]",
                     "diff_fp32_min": -0.5,
                     "diff_fp32_max": 1
                 },
-                "mobilenet_v2_int8_w_sym_t_half_a_sym_t_rb_sparsity_52": {
+                "mobilenet_v2_imagenet_rb_sparsity_int8": {
                     "config": "examples/tensorflow/classification/configs/sparsity_quantization/mobilenet_v2_imagenet_rb_sparsity_int8.json",
-                    "reference": "mobilenet_v2",
-                    "target": 71.0,
+                    "reference": "mobilenet_v2_imagenet",
+                    "target": 70.96,
```

Comment:
If you are going to update the target metric every release, then we may end up in a situation where the metric will change significantly.

Сan we not change target metrics in the automatic flow?

Reply:
The FP32 metric can only change if the upstream FP32 model is retuned or improved. If this leads to our existing checkpoints having less than 99% FP32, it will show. The entire script in `tools/update_eval_results.py` is only meant to be run manually, based off E2E results that we find acceptable in their entirety.